src/models/relayer/rpc_config.rs

```diff
@@ -100,7 +128,7 @@ mod tests {
     fn test_with_weight_creates_config_with_custom_weight() {
         let url = "https://example.com".to_string();
         let weight: u8 = 5;
-        let config = RpcConfig::with_weight(url.clone(), weight);
+        let config = RpcConfig::with_weight(url.clone(), weight).unwrap();
```

Comment:
Let's map error to RpcConfigError

using unwrap could crash server with panics